packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/configmid.md

```diff
@@ -0,0 +1,25 @@
+The Okta Angular SDK requires an instance of an `OktaAuth` object with configuration properties. You should set the `ClientID` and `Issuer` properties. This can happen by directly setting the properties, with variable replacement that happens as part of the build process, or during application load time.
+
+Add `OktaAuthModule` to the `NgModule` `imports` array and configure the Okta provider so Okta services are added to the dependency injection system. The Okta Angular SDK provides the `OKTA_CONFIG` injection token where you'll use the `OktaAuth` instance as the value.
```

Comment:
```suggestion
Add `OktaAuthModule` to the `NgModule` `imports` array and configure the Okta provider so that Okta services are added to the dependency injection system. The Okta Angular SDK provides the `OKTA_CONFIG` injection token where you use the `OktaAuth` instance as the value.
```